Kernel/API/ttydefaultschars.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2021, Daniel Bertalan <dani@danielbertalan.dev>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <Kernel/API/POSIX/termios.h>
+#include <Kernel/API/ttydefaults.h>
+
+#ifdef __clang__
+#    pragma clang diagnostic push
+#    pragma clang diagnostic ignored "-Wc99-designator"
+#endif
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+static const cc_t ttydefchars[NCCS] = {
```

Comment:
I worry about this change. Is the default chars some BSD compat thing? The macro made it seem like it is. 

Are the constants Posix? Perhaps the file should be moved to API/POSIX

Reply:
Oh wait. Never mind me you kept that in the C header


In that case, the ifdefs for kernel in the LibC header can be removed